scripts/circleci/run-ami-tests-in-bg.sh

```diff
@@ -21,14 +21,23 @@ echo "Runing AMI sanity tests concurrently in the background (this may take up t
 # Create directory which output log files will be saved
 mkdir -p outputs
 
-# run sanity test for each image concurrently in backgroung.
-python tests/ami/packages_sanity_tests.py --distro=WindowsServer2012 --type=install --to-version=/tmp/workspace/ScalyrAgentInstaller.msi &> outputs/WindowsServer2012.log &
-python tests/ami/packages_sanity_tests.py --distro=WindowsServer2016 --type=install --to-version=/tmp/workspace/ScalyrAgentInstaller.msi &> outputs/WindowsServer2016.log &
-python tests/ami/packages_sanity_tests.py --distro=WindowsServer2019 --type=install --to-version=/tmp/workspace/ScalyrAgentInstaller.msi &> outputs/WindowsServer2019.log &
-python tests/ami/packages_sanity_tests.py --distro=ubuntu1804 --type=upgrade --from-version=current --to-version=/tmp/workspace/scalyr-agent-2.deb &> outputs/ubuntu1804.log &
-python tests/ami/packages_sanity_tests.py --distro=ubuntu1604 --type=upgrade --from-version=current --to-version=/tmp/workspace/scalyr-agent-2.deb &> outputs/ubuntu1604.log &
-python tests/ami/packages_sanity_tests.py --distro=ubuntu1404 --type=upgrade --from-version=current --to-version=/tmp/workspace/scalyr-agent-2.deb &> outputs/ubuntu1404.log &
-python tests/ami/packages_sanity_tests.py --distro=centos7 --type=upgrade --from-version=current --to-version=/tmp/workspace/scalyr-agent-2.rpm &> outputs/centos7.log &
+# Run sanity test for each image concurrently in background
+# Tests below utilize installer script to test installing latest stable version of the package
+python tests/ami/packages_sanity_tests.py --distro=ubuntu1804 --type=install --to-version=current --installer-script-url="https://www.scalyr.com/scalyr-repo/stable/latest/install-scalyr-agent-2.sh" &> outputs/ubuntu1804-install.log &
```

Comment:
@Kami Do we really need to check the stable release from the repo here? I thought the idea is that the stable release gets its own portion of tests when we merge changes to master and release branches?

Reply:
Yeah, one option is to split it out in a separate job, since ideally we would also run those stable install tests periodically (e.g. every 24 hours).

Reply:
And to expand on that - doing that (adding a new job for running install tests using installer script and stable packages) would give us flexibility to run that job periodically as well (e.g. every 24 hours which is a good idea), but it does increase complexity quite a bit (new Circle CI job, additional script to run those tests).

Reply:
This has now been resolved - thanks for your assistance with #574.